woocommerce/woocommerce-ios
# Dark Mode: color updates for Edit Product and Aztec editor

Fixes #1524 

## Changes

- In `AztecUIConfigurator`, set the text color of the rich text view
- In `AztecFormatBarFactory`, set the background color of the formatting toolbar above the keyboard
- In `AddProductImageCollectionViewCell`, used `UIColor` instead of `CGColor` to avoid caching. Also updated the border color when Light/Dark mode changes
- Added optional `imageTintColor` to `ImageAndTitleAndTextTableViewCell.ViewModel` for supporting image color in Light/Dark mode changes. Updated Edit Product inventory/price/shipping icons to use `imageTintColor` instead of tinting the image color in a static way
- Added a semantic color for placeholder image tint color `placeholderImage` and used it for Aztec placeholder image

## Testing

- Launch the app
- Go to Products tab
- Tap on a Product
- Tap "Edit" in the navigation bar --> the colors should look as expected in Light and Dark modes
- Tap the Product description row --> the Aztec editor should look as expected in Light and Dark modes

## Example screenshots

@kyleaparker I wasn't sure whether to use the accent color for the Aztec editor toolbar. feel free to suggest other colors!

screen | Dark | Light
-- | -- | --
Edit Product | ![Simulator Screen Shot - iPhone 8 - 2019-12-30 at 10 24 41](https://user-images.githubusercontent.com/1945542/71566171-49638d00-2af0-11ea-984c-791e95ba7d13.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-30 at 10 24 49](https://user-images.githubusercontent.com/1945542/71566172-49fc2380-2af0-11ea-8593-4ed00ca5fff2.png)
Aztec editor | ![Simulator Screen Shot - iPhone 8 - 2019-12-31 at 12 29 18](https://user-images.githubusercontent.com/1945542/71610109-307ada80-2bc9-11ea-98a1-a0903b978e6c.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-31 at 12 29 09](https://user-images.githubusercontent.com/1945542/71610108-307ada80-2bc9-11ea-9be2-9f255d0bc8ac.png)


Update release notes:
